File: src/main/java/com/noahhendrickson/elefant/commands/other/JerCommand.java
package com.noahhendrickson.elefant.commands.other;

import com.noahhendrickson.elefant.CommandBundle;
import com.noahhendrickson.elefant.ICommand;

/**
 * Created by Noah Hendrickson on 4/12/2020
 */
public class JerCommand implements ICommand {

    @Override
    public void execute(CommandBundle bundle) {
        bundle.sendMessage("<:java:585593753629229081>", "Learn Java First!");
    }

    @Override
    public String getCommand() {
        return "jer";
    }

    @Override
    public String getDescription() {
        return "A jer thing to say.";
    }
}

File: src/main/kotlin/com/noahhendrickson/commands/JerCommand.kt
package com.noahhendrickson.commands

import com.noahhendrickson.elefant.commands.Command
import net.dv8tion.jda.api.events.message.MessageReceivedEvent

class JerCommand : Command() {

    override fun execute(args: Array<out String>?, event: MessageReceivedEvent?) {
        event?.message?.delete()?.queue()
        super.sendMessage("Maybe you should learn java fist before making Minecraft plugins...", event)
    }

    override fun getAliases(): MutableList<String> {
        return mutableListOf("!jer", "!learnjava")
    }

    override fun getName(): String {
        return "Learn Java"
    }

    override fun getDescription(): String {
        return "Learn Java first."
    }

    override fun getUsageInstructions(): MutableList<String> {
        return mutableListOf("")
    }
}